articles/firewall/firewall-faq.yml

```diff
@@ -146,8 +146,8 @@ sections:
           When you allocate and deallocate, [firewall billing](https://azure.microsoft.com/pricing/details/azure-firewall) stops and starts accordingly.
 
           > [!NOTE]
-          > You must reallocate a firewall and public IP to the original resource group and subscription.
-
+          > You must reallocate a firewall and public IP to the original resource group and subscription. 
+          > When stop/start is performed, the private IP address of the firewall may change to a different IP addresss within the subnet.  This can effect connectivity of previously configured route tables. 
```

Comment:
```suggestion
          > When stop/start is performed, the private IP address of the firewall may change to a different IP address within the subnet.  This can affect the connectivity of previously configured route tables. 
```